gems/ibm_cloud_power/lib/ibm_cloud_power/version.rb

```diff
@@ -11,5 +11,5 @@
 =end
 
 module IbmCloudPower
-  VERSION = '1.0.0'
+  VERSION = '1.2.0'
```

Comment:
I had to update this in order to test, but am not sure if I should remove it and release with a separate commit after this PR is merged.

Reply:
I'm fine with this as part of the PR since I don't expect to be releasing a number of changes in a version bump here

Reply:
Ideally the power cloud openapi spec would maintain proper versioning and this would be auto-updated along with everything else.